File: cellab_devkit-1.0 (1)/rules_java/wind.java
/*

        Wind tunnel skeleton rule
        
        No lookup table is used.

*/

class Wind extends ruletable {
    void jcruleModes() {
        setWorld(12);                   // User evaluator, open world
        setOwnCodeRequest("cfd");       // User evaluator function
        setPatternRequest("wind");
        setPaletteRequest("wind");
    }

    int jcrule(int oldstate) {
        return 0;               // No lookup table
    }
}

public class wind {
    public static void main(String args[]) {
        (new Wind()).generateRuleFile("wind");
    }
}

File: references/cellab_devkit-1.0 (1)/ruledefs/wind.js
    /*

        Wind tunnel skeleton rule
        
        No lookup table is used.

    */

    rule.worldtype = 12;        // 2D open world, user evaluator
    rule.patreq = "wind";
    rule.palreq = "wind";
    rule.ocodereq = "cfd";

    function owncode(oldstate) {
        return 0;               // No lookup table
    }
